crates/octez/Cargo.toml

```diff
@@ -27,5 +27,4 @@ signal-hook.workspace = true
 tempfile.workspace = true
 tezos_crypto_rs.workspace = true
 tezos-smart-rollup-encoding.workspace = true
-tezos_crypto_rs.workspace = true
```

Comment:
removing this as it wasn't raised as a conflict and thus was missed out when #610 was merged